lib/src/_shared/hooks/usePickerState.ts

```diff
@@ -62,17 +63,19 @@ export function usePickerState(props: BasePickerProps, options: StateHookOptions
       onClear: () => acceptDate(null),
       onAccept: () => acceptDate(pickerDate),
       onSetToday: () => setPickerDate(utils.date()),
-      onDismiss: () => {
-        setIsOpen(false);
-      },
+      onDismiss: () => setIsOpen(false),
```

Comment:
Haha, we have two schools of thought. I appreciate the previous approach as when we refactor, it's obvious the method doesn't return anything and we can move on. On the other hand, the new approach is shorter.

Reply:
I am always using the shorter way because the code itself in typescript and I always know if method returns `void` or not